apps/explorer/src/ui/SenderRecipient.tsx

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { Amount } from '~/ui/Amount';
+import { Heading } from '~/ui/Heading';
+import { SenderRecipientAddress } from '~/ui/SenderRecipientAddress';
+
+type Recipient = {
+    address: string;
+    coin?: {
+        amount: number;
+        coinSymbol?: string;
```

Comment:
Should just be `symbol`. It's also somewhat unclear to me when this is optional.